app/Http/Controllers/CompetitionsController.php

```diff
@@ -37,18 +37,6 @@ public function __construct(UserRepositoryContract $repository, Manager $manager
     }
 
     /**
-     * Display a listing of the resource.
-     *
-     * @return \Illuminate\Http\Response
-     */
-    public function index()
```

Comment:
> remove all competitions link from the nav

Figured it would need to go because we're taking out of the Nav. Would the "want to see all comps I'm running" be a diff route?